v3/src/components/graph/graph.scss

```diff
@@ -0,0 +1,149 @@
+.plot {
+  width: 100%;
+  height: 100%;
+  background-color: aliceblue;
+}
+
+.graph-svg {
+  width: 100%;
+  height: 100%;
+  margin: 25px;
+}
+
+.axis {
+  background-color: white;
+}
+
+.plot text {
+  fill: black;
+  font: 10px sans-serif;
+}
+
+.dotArea {
+  background: white;
+}
+
+.background {
+  fill: white;
+}
```

Comment:
Generally, .scss should be as close to its code as possible, which generally means associated with particular components where possible, and often nested within each other which minimizes the chances of global conflicts. That way, if the case table also wants to use `.background` as a class name the two definitions won't conflict with each other.